messages/pt.json

```diff
@@ -6,6 +6,9 @@
   "store/checkout-summary.Shipping": "Entrega",
   "store/checkout-summary.Total": "Total",
   "store/checkout-summary.disclaimer": "Taxas e frete calculados no Carrinho",
+  "store/checkout-summary.installments.default": "Em até {installmentsNumber}x {installmentValue}{hasInterest, select, true {, {interestRate} de juros} false { sem juros}}",
+  "admin/editor.checkout-summary.installments.description": "Valores disponiveis para interpolação: '{installmentsNumber}, {installmentValue}, {installmentsTotalValue}, {interestRate}, {hasInterest}, {paymentSystemName}'",
```

Comment:
"admin/editor.checkout-summary.installments.description": "Valores disponíveis para interpolação: '{installmentsNumber}, {installmentValue},

Reply:
Hello @camilavcoutinho !

@jeffersonacct made the code changes, could you validate, please?